prow/upgrade-tests.sh

```diff
@@ -45,12 +45,16 @@ download_untar_istio_release "${TARGET_RELEASE_PATH}" "${TARGET_VERSION}"
 
 # Check https://github.com/istio/test-infra/blob/master/boskos/configs.yaml
 # for existing resources types
-export RESOURCE_TYPE="${RESOURCE_TYPE:-gke-e2e-test}"
-export OWNER='upgrade-tests'
-export USE_MASON_RESOURCE="${USE_MASON_RESOURCE:-True}"
-export CLEAN_CLUSTERS="${CLEAN_CLUSTERS:-True}"
-
-setup_e2e_cluster
+if [ -z "${UPGRADE_TEST_LOCAL}" ]; then
```

Comment:
instead of adding a new one, you should be able to reuse https://github.com/istio/istio/blob/df0d817c5ecf6f3fab94cb599ddd90c363c4d533/prow/lib.sh#L91 instead.

Reply:
done.

Reply:
I mean USE_MASON_RESOURCE should be able to take care of what you need. You should not need ${UPGRADE_TEST_LOCAL}

Reply:
I tried it but then had to revert since it didn't work.